samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.h

```diff
@@ -26,6 +26,10 @@
 #define CANNOT_SET_RANGE_MIN		0x01
 #define CANNOT_SET_RANGE_MAX		0x02
 
+#define ERROR 0x00
```

Comment:
Use an enum without explict values if these don't map to some specification-defined values (looks like they're purely internal to your application). Also give them more specific names (perhaps SET_ERROR, SET_ON, SET_OFF) since what you use now have a high probability of conflicting with some other defines in the system.

Reply:
It is not related to state ..so I can't use SET_ERROR, SET_ON, SET_OFF.

Goal is state binding algorithm should not initiate if Gen. OnOff Sevrer state is already equal to 1 & client too set value as 1.

Reply:
Assume current lightness = 10 % & default lightness = 50% then if client send OnOff = 1 then that initiate state binding & changes lightness state from 10 % to 50 %.

To avoid that I used three values,
0 ... means error
1 .... means initiate state binding + send GET status + publish the message for Model's publisher address
2.... means  just send GET status + publish the message for Model's publisher address

Reply:
Delete last commit as of now. I will take care of it in next PR.